apps/system/js/nfc_handover_manager.js

```diff
@@ -276,12 +264,11 @@
      * @memberof NfcHandoverManager.prototype
      */
     _doAction: function _doAction(action) {
-      if (!this.bluetooth.enabled) {
+      if (!Service.query('Bluetooth.isEnabled')) {
         this._debug('Bluetooth: not yet enabled');
         this.actionQueue.push(action);
         if (this.settingsNotified === false) {
-        // XXX: Let BluetoothCore do this for you
-          this.writeSetting({'bluetooth.enabled': true});
+          window.dispatchEvent(new CustomEvent('request-enable-bluetooth'));
```

Comment:
Same here, this.publish('request-enable-bluetooth', this, /* no prefix */ true);